package.json

```diff
@@ -68,7 +70,7 @@
     "react-native-video": "^4.4.1",
     "react-native-video-controls": "^2.2.3",
     "react-native-webview": "5.8.1",
-    "react-navigation": "^3.9.1",
+    "react-navigation": "^3.11.0",
```

Comment:
Do you really need to update?

Reply:
Because the ```safe-area-view``` -> ```react-navigation``` is a dependency on 'react-native-safe-area-view', in iOS extensions there was an issue in the safe-area-view but this has been fixed in the newer version.  https://github.com/react-native-community/react-native-safe-area-view/issues/66